docs/synapse/userguides/model_updates/update_v2_181_0.rst

```diff
@@ -0,0 +1,88 @@
+
+
+.. _userguide_model_v2_181_0:
+
+######################
+v2.181.0 Model Updates
+######################
+
+The following model updates were made during the ``v2.181.0`` Synapse release.
+
+*********
+New Forms
+*********
+
+``lang:phrase``
+  A small group of words which stand together as a concept.
+
+
+``math:algorithm:type:taxonomy``
+  A hierarchical taxonomy of algorithm types.
+
+
+``ou:industry:type:taxonomy``
+  An industry type taxonomy.
+
+
+``math:algorithm``
+  A mathematical algorithm.
+
+
+
+**************
+New Properties
+**************
+
+``ou:campaign``
+  The form had the following property added to it:
+
+  ``slogan``
+    The slogan used by the campaign.
+
+
+``ou:org``
+  The form had the following property added to it:
+
+  ``motto``
+    The motto used by the organization.
+
+
+``risk:vulnerable``
+  The form had the following properties added to it:
+
+
+  ``mitigated``
+    Set to true if the vulnerable node has been mitigated.
+
+
+  ``mitigations``
+    The mitigations which were used to address the vulnerable node.
+
+
+
+***********
+Light Edges
+***********
+
+``generates``
+    When used with a ``math:algorithm`` node, the edge indicates the target
+    node was generated by the algorithm.
+
+
+``has``
+    When used with a ``it:dev:repo`` and an ``inet:url`` node, the edge
```

Comment:
```suggestion
    When used with an ``it:dev:repo`` and an ``inet:url`` node, the edge
```

Reply:
'a' before consonant sound, 'an' before vowel sound. Adjusting the generator to use 'an' before a vowel character would probably get it right most of the time.

Reply:
I'm tracking that change separately.